functions/src/bdd/entityHandlers/RoomHandler.ts

```diff
@@ -0,0 +1,27 @@
+import {EntityHandler} from "./EntityHandler";
+import {Room} from "../entities";
+import {EntityManager} from "@mikro-orm/mysql";
+import { castToRoomData } from "../../types/request/bodyData";
+
+export class RoomHandler extends EntityHandler {
+    constructor(entityManager: EntityManager) {
+        super(entityManager, Room);
+    }
+
+    async createRoom(payload: any) {
+        const roomData = castToRoomData(payload);
+
+        if(roomData === null) return null;
```

Comment:
```suggestion
        roomData ?? null;
```

À essayé, mais du coup en faisait ça, c'est soit tu retourne roomData s'il existe, soit null, c'est ça que tu voulais faire ?

Reply:
Le but n'est pas de retourner roomData car il sert par la suite. Le but du Create reste de créer une Room dans la globalité, et là c'est juste transformer les data pour les cast vers le type RoomData ( permettant de vérifier les data )